ansible/examples/slave_libvirt.yml

```diff
@@ -74,7 +73,7 @@
       become_user: "{{ jenkins_user }}"
 
     - name: set the authorized keys
-      authorized_key: user={{ jenkins_user }} key="{{ lookup('file', 'files/ssh/keys/jenkins_build.pub') }}"
+      authorized_key: user={{ jenkins_user }} key="{{ lookup('file', 'playbook/files/ssh/keys/jenkins_build.pub') }}"
```

Comment:
ah this might be specific to prado then, I will add it back to examples (but not to the top level yaml files)